lib/seattleflu/db/cli/command/etl/manifest.py

```diff
@@ -185,7 +185,8 @@ def upsert_sample(db: DatabaseSession,
     # Look for existing sample(s)
     with db.cursor() as cursor:
         cursor.execute("""
-            select sample_id as id, identifier, collection_identifier, encounter_id
+            select sample_id as id, identifier, collection_identifier,
+                encounter_id, details
```

Comment:
Is this necessary? I thought this was just looking up the sample and it gets updated later?

Reply:
The details column has to be selected for update, or the existing samples won't receive the new data we're pulling from the manifest data.

Reply:
Oh, you're right. I thought that I couldn't update details without selecting it here, but I just tested it again, and it doesn't seem necessary.